src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -49,8 +49,6 @@ class PeerActor(
 
   val peerId: String = self.path.name
 
-  val daoForkValidator = ForkValidator(daoForkBlockNumber, daoForkBlockHash)
```

Comment:
Should the `ForkValidator` and `MessageValidator` classes be removed as they are no longer used?